extensions/rich_text_components/Skillreview/webdriverio.js

```diff
@@ -30,9 +30,10 @@ var customizeComponent = async function(modal, text, skillDescription) {
 
   var textEditor = objects.UnicodeStringEditor(
     modal.$('<schema-based-unicode-editor>'));
-  // Do not change the text if there is already text present as this
-  // RTE component sometimes relies on highlighting text.
-  if (textEditor.getValue() && !text) {
+  // Change the text only if it is not already present or if
+  // the text argument is specified.
+  if ((textEditor.getValue() && text !== '') ||
+      (text === null || text === undefined)) {
```

Comment:
This still doesn't seem right. If the text editor has some content (so `textEditor.getValue()` is truthy) and `text` is non-empty (so `text !== ''` is true), then a user will probably expect the existing text to be over-written by the new text, but this code will cause the function to `return` without changing the text.

I think the only time the text should be unchanged is when `text` is null or `text === textEditor.getValue()`, right?

Reply:
Yes that sounds right, this was a slightly old commit though I changed it to `(text === null || text === undefined)`, but I think `text === null || text === textEditor.getValue()` is better.
Done